packages/database/src/models/AccessLog.ts

```
@@ -0,0 +1,81 @@
+  declare id: string;
+  declare userId: string;
+  declare recordId: string;
+  declare facilityId?: string;
+  declare sessionId: string;
+  declare loggedAt: string;
+  declare frontEndContext: string;
+  declare backEndContext: string;
+  declare isMobile: boolean;
+
+  static initModel({ primaryKey, ...options }: InitOptions) {
+    super.init(
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)